### DISCUSSION TITLE
Other AI providers proposal

### DISCUSSION THREAD

**Discussion by pfrankov** [pfrankov/obsidian-local-gpt#1] [Ideas]
If you know easy-to-use alternatives for Ollama and want to use it in the plugin, follow the template:
- Name and link
- Supported operating systems (Win, Mac, Linux, iOS, Android)
- Describe, what's the main difference from Ollama? In other words, why it should be added in the plugin.

**Comment by Linatree** [comment]
A great plug-in, especially the ability to freely add keys and modify prompt words. I use it very well. Please add the API interfaces of gemini and claude so that I can freely switch language models without having to install other AI plug-ins. Thank you:)

**Reply by wangy4755** [reply]
Echo on the Claude and Gemini, there are other plug-i provide these support, but none is like yours. clean, simple, straightforward and easy to use.

**Comment by insujeon** [comment]
What do you think about Google Gemini's API support? Since the plug-in currently only supports the Openai-compatible API, it would be great to enable Google's.

**Reply by pfrankov** [reply]
🎉 Just added Gemini support in AI Providers (update plugins in Obsidian settings)
